benchmark_runner/common/template_operations/templates/hammerdb/internal_data/mssql_template.yaml

```diff
@@ -71,31 +112,18 @@ spec:
             value: "Y"
           - name: MSSQL_SA_PASSWORD
             value: "s3curePasswordString"
-        {%- if odf_pvc == True %}
+        securityContext:
+          allowPrivilegeEscalation: true
+      {%- if odf_pvc == True %}
```

Comment:
Unless I misunderstood, the below volumeMounts/volumes are not defined at all for LSO mode?

Reply:
You right, I run LSO always with odf_pvc == True.
I have fixed it in all the database:
`{%- if storage_type == 'lso' or odf_pvc == True %} `